### ISSUE TITLE
prefer-compact should warn if filter uses default identity function

### ISSUE THREAD

**Issue by martinmacko47** [wix/eslint-plugin-lodash#177]
As `prefer-compact` considers an explicit identity function a warning:

```
_.filter(arr, (x) => x)
```

It should consider the implicit one a warning as well, shouldn't it?

```
_.filter(arr)
```

**Comment by ganimomer** [comment]
You're right, it should.

**Comment by ganimomer** [comment]
Fixed in version `2.6.1`. Thanks for opening the issue!
